kinoml/modeling/homology.py

```diff
@@ -43,7 +45,7 @@ def get_pdb_template(
         return hits
 
     def get_sequence(
-        self, identifier: str, kinase: bool = False, backend: str = "uniprot"
+        self, identifier: str, kinase: bool = True, backend: str = "uniprot"
```

Comment:
I am not sure about this kind of behaviour change, but I'll review this in the context of making all the structure-based PRs consistent with each other. No need for action now, this is just a note to my future self :)